### ISSUE TITLE
sk.php get wiped during upgrade to 2.1.27

### ISSUE THREAD

**Issue by jerheij** [nilsteampassnet/TeamPass#1798]
### Steps to reproduce
1. Run teampass <2.1.27
2. Attempt upgrade to 2.1.27
3. Copy existing sk.php
4. Run upgrade.php wizard

### Expected behaviour
Upgrade shouldn't wipe sk.php.

### Actual behaviour
Upgrade wipes sk.php which results in passwords inaccessible after the upgrade.

### Server configuration
**Operating system**:
Centos 7
**Web server:**
HTTPD
MySQL version 
5.1.73
PHP version 
5.6.30
Teampass version 
2.1.26.14 / 2.1.26-final-3
**Updated from an older Teampass or fresh install:**
Update to 2.1.27.x from 2.1.26.14
### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
```
Insert your webserver log here
```

#### Firebug log ([How to?](http://teampass.net/2014-02-09-how-to-communicate-an-error-log))
```
Insert the Firebug log here
```

**Edit:**
Changed the versions to the right value.

**Comment by nilsteampassnet** [comment]
I'm surprised as there is no instruction in the code to delete the sk.php.
At contrario, it creates a copy of this file before being adapted.

What package do you use? From what path did you download it?

**Comment by jerheij** [comment]
The sk.php file wasn't deleted, the content was altered. The line containing the salt key was removed from the php file.

I wasn't able to read any of the password item information after the upgrade and then I noticed that the salt wasn't there any more. I was on version 2.1.26.14 at first but it also happened on version 2.1.26-final-3. I downloaded all of them from your github page, first the "master.zip" and then different 2.1.27 releases:
- 2.1.27.4
- 2.1.27.0

I was able to solve this by executing a "chmod 750" on the sk.php so the webserver couldn't change the content, after that the upgrade succeeded and I could access all my passwords again.
